clang-tools-extra/clangd/XRefs.cpp

```diff
@@ -1751,6 +1752,7 @@ static std::optional<HierarchyItem> symbolToHierarchyItem(const Symbol &S,
   }
   HierarchyItem HI;
   HI.name = std::string(S.Name);
+  HI.detail = (S.Scope + S.Name).str();
```

Comment:
adding signature looks a bit more helpful, maybe one day add return type info too
```suggestion
  HI.detail = (S.Scope + S.Name + S.Signature).str();
```

Reply:
I agree, showing the signature here would be quite useful.

I actually started implementing this for incoming calls in https://github.com/clangd/clangd/issues/915, but ran into a complication (our current indexing process doesn't actually store the signature for all functions).

This is solvable with some indexer changes, but I'll defer to a fix for that issue, which will now (once this patch is merged) benefit both incoming and outgoing calls.